docs/source/DEVELOPER.rst

```diff
@@ -154,7 +154,7 @@ The DeepCell Kiosk comes with a utility for benchmarking the scalability and per
 
 2. There are three variables in the benchmarking pod's YAML file, ``conf/helmfile.d/0410.benchmarking.yaml``, that may need to be customized before benchmarking:
 
-    - ``MODEL`` is the model name and version that will be used in benchmarking. The model you choose should be present in the ``models/`` folder of your benchmarking bucket. See the `Van Valen Lab's benchmarking bucket <https://console.cloud.google.com/storage/browser/kiosk-benchmarking>`_ for an example.
+    - ``MODEL`` is the model name and version that will be used in benchmarking. The model you choose should be present in the ``models/`` folder of your benchmarking bucket. See the `Van Valen Lab's benchmarking bucket <https://storage.googleapis.com/kiosk-benchmarking>`_ for an example.
```

Comment:
```suggestion
    - ``MODEL`` is the model name and version that will be used in benchmarking. The model you choose should be present in the ``models/`` folder of your benchmarking bucket. See the `Van Valen Lab's benchmarking bucket <https://console.cloud.google.com/storage/browser/kiosk-benchmarking>`_ for an example.
```

Actually, the last link worked, but the problem was on the bucket side. I had only given read access to particular files, so the link to the bucket itself did not work. I've fixed the read-access and now the former link should work.